app/views/pages/home/_location_categories.html.haml

```diff
@@ -1 +1,14 @@
-%h1.govuk-heading-xl Location Categories
+%h1.govuk-heading-xl #{ t('.location_links_heading') }
```

Comment:
this should be
`%h1.govuk-heading-l` not xl

Reply:
Done 👍